inferno-ml/src/Inferno/ML/Module/Prelude.hs

```diff
@@ -180,6 +183,8 @@ module ML
 
   softmax : int -> tensor -> tensor := ###softmaxFun###;
 
+  stack : int -> array of tensor -> tensor := ###stackFun###;
```

Comment:
You could add a docstring to this function definition so that uses know what it does and how to use it. See for example `argmax` above.

Reply:
Will do